openpype/hosts/blender/utility_scripts/make_paths_relative.py

```diff
@@ -23,8 +24,8 @@
                     str(Path(datablock.filepath).resolve()),
                     start=str(Path(bpy.data.filepath).parent.resolve()),
                 )
-        except (RuntimeError, ReferenceError, ValueError, OSError) as e:
-            log.error(e)
+        except BaseException as e:
+            raise SystemExit(e)
 
     bpy.ops.file.make_paths_relative()
     bpy.ops.wm.save_mainfile()
```

Comment:
Can we include this final ops in a try except as above ?

Reply:
Sure!